app/src/main/java/com/haruta/harutyan/countchallenge/MainActivity.kt

```diff
@@ -21,10 +23,20 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater).apply { setContentView(this.root) }
 
+        var saveNumber: SharedPreferences = getSharedPreferences("SharedPref", Context.MODE_PRIVATE)
```

Comment:
変数の中身を書き換えたくないので、 `val` にしたい…！

また、変数の名前が saveNumber のように「動詞+名詞」だと関数のように見えてしまうので、名詞で命名するのが良さそう！
今回に関しては、SharedPreferences では Number 以外も保存する可能性があるので、 `sharedPreferences` の方が良さそう！
```suggestion
        val sharedPreferences: SharedPreferences = getSharedPreferences("SharedPref", Context.MODE_PRIVATE)
```